modules/schematics/src/ng-add/index.ts

```diff
@@ -1,40 +1,37 @@
 import {
   chain,
-  noop,
   Rule,
   SchematicContext,
   Tree,
 } from '@angular-devkit/schematics';
 import { getWorkspace, getWorkspacePath } from '../../schematics-core';
 
-import { Schema as SchematicOptions } from './schema';
-
-function updateWorkspaceCli(host: Tree, value: any) {
+function updateSchematicCollections(host: Tree) {
   const workspace = getWorkspace(host);
   const path = getWorkspacePath(host);
 
-  workspace['cli'] = {
-    ...workspace['cli'],
-    ...value,
-  };
+  if (!(workspace['cli'] && workspace['cli']['schematicCollections'])) {
+    throw new Error(
```

Comment:
Why are we throwing an error here? If the schematicCollections property doesn't exist, we should create it and add @ngrx/schematics to it.

Reply:
That would be on me 😅
https://github.com/ngrx/platform/issues/3383#issuecomment-1140199497

Reply:
Since it's 2v1, I'm fine with creating the property.
I don't know what Angular's schematic will do with it though (in the case that NgRx is updated before Angular)

Reply:
Ahh ok. I was thinking of cases like Nx where it may not be present, but I think they account for that anyway. Throwing the error should be ok for now